SQL JOINs

mihaiberq opened the pull request:
- 👶💪JOINs
  - [x] INNER JOIN
  - [x] LEFT / RIGHT JOIN
  - [x] FULL JOIN
  - [x] Multiple JOINs

enki-hq commented:
- [ ] https://enkipro.com/preview-insight/sagelabs/content/SQL/DQL/joins/full-join.md?ref=0c3a7fb8bb57923979f6d700c68aec5c8f3b8d4f
- [ ] https://enkipro.com/preview-insight/sagelabs/content/SQL/DQL/joins/inner-join.md?ref=0c3a7fb8bb57923979f6d700c68aec5c8f3b8d4f
- [ ] https://enkipro.com/preview-insight/sagelabs/content/SQL/DQL/joins/left-and-right-joins.md?ref=0c3a7fb8bb57923979f6d700c68aec5c8f3b8d4f
- [ ] https://enkipro.com/preview-insight/sagelabs/content/SQL/DQL/joins/multiple-joins.md?ref=0c3a7fb8bb57923979f6d700c68aec5c8f3b8d4f

mihaiberq commented:
@lizTheDeveloper Sorry about that, changed master to `sql` branch and after I opened the PR and forgot to double check. Moved everything to this one.

lizTheDeveloper commented on SQL/DQL/joins/full-join.md:
```diff
@@ -0,0 +1,72 @@
+---
+## Practice
+
+The `FULL OUTER JOIN` operations is equivalent to
```
have users do a full outer join with fill in the gaps

lizTheDeveloper commented on SQL/DQL/joins/full-join.md:
````diff
@@ -0,0 +1,72 @@
+FROM pokemon_type
+... type ON
+pokemon_type.type_id = type.id
+```
````
do a mulit-part gap fill, add the ON part into the ??? slots

lizTheDeveloper commented on SQL/DQL/joins/inner-join.md:
```diff
@@ -0,0 +1,92 @@
+---
+## Footnotes
+
+[1: Ability]
```
💯  for accurate footnotes

lizTheDeveloper commented on SQL/DQL/joins/inner-join.md:
```diff
@@ -0,0 +1,92 @@
+---
+## Revision
+
+Fill in the blanks such that the following snippet contains a valid JOIN operations:
```
switch the practice and revision question

lizTheDeveloper commented on SQL/DQL/joins/multiple-joins.md:
```diff
@@ -0,0 +1,81 @@
+
+In the Pokémon database you can find multiple intermediate tables, usually named `table1_table2`: for the `pokemon` and `type` tables, the intermediate one is named `pokemon_type`. Same goes for `pokemon`, `move` and `pokemon_move`.
+
+A single JOIN is simple to achieve: take a table, use the magic word (`type JOIN`), specify the second table and choose a joining field. However, you can also join multiple tables together.  
```
you don't need this sentence

lizTheDeveloper commented on SQL/DQL/joins/multiple-joins.md:
```diff
@@ -0,0 +1,81 @@
+
+---
+## Practice
+
```
make this a fill-the-gaps question
